### ISSUE TITLE
Hi, we have recently updated a new VisDrone training config (gradient clip added), which can avoid NaN now.

### ISSUE THREAD

**Issue by shaoyandea** [ChenhongyiYang/QueryDet-PyTorch#97]
Hi, we have recently updated a new VisDrone training config (gradient clip added), which can avoid NaN now.

_Originally posted by @ChenhongyiYang in https://github.com/ChenhongyiYang/QueryDet-PyTorch/issues/40#issuecomment-1433519429_
            I also meet this problem ,have you solve it?

**Comment by ChenhongyiYang** [comment]
Hi, may I ask what is the total batch size in your training?

**Comment by shaoyandea** [comment]
I trained with coco datadet, the total batch size&nbsp; is 12, and I have solved Nan by adding gradient clip set as visdrone_train.yaml, also the reproduced result is close to result in paper. there is a bug that learning rate always keep original setting in whole training, and iters in reume training is wrong.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ChenhongyiYang/QueryDet-PyTorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月4日(星期二) 晚上11:00
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ChenhongyiYang/QueryDet-PyTorch] Hi, we have recently updated a new VisDrone training config (gradient clip added), which can avoid NaN now. (Issue #97)





 
Hi, may I ask what is the total batch size in your training?
